pkg/symbol/symbol.go

```diff
@@ -71,25 +81,43 @@ func (s *Symbolizer) Symbolize(prof *profiler.Profile) error {
 		}
 	}
 
+	pid := prof.ID.PID
+
 	if s.vdsoCache != nil {
 		for _, l := range prof.UserLocations {
-			if l.Mapping.File == "[vdso]" {
-				name, err := s.vdsoCache.Resolve(l.Address, l.Mapping)
+			if l.Mapping.File != "[vdso]" {
```

Comment:
no need to change the logic here.
the package vdso expect unomarlized address, ant itself would always do the normalization.
currently the vdso mapping file is [vdso], the functoin objectfile.ObjAddr() could not normalized vdso address, so the package vdso would always get unnomarlized address.

Reply:
I see, thank you!